test-report/htmlWptReport.mjs

```diff
@@ -30,101 +31,7 @@ function getOutputPath() {
   return process.argv.slice(2)[1];
 }
 
-function escapeHtml(str) {
-  return str
-    .replace(/&/g, '&amp;')
-    .replace(/</g, '&lt;')
-    .replace(/>/g, '&gt;')
-    .replace(/"/g, '&quot;')
-    .replace(/'/g, '&#39;')
-    .replace(/\//g, '&#47;');
-}
-
-function generateHtml(report) {
-  const stat = {all: 0, pass: 0};
-  for (const test of report.results) {
-    if (test.status !== 'OK' && test.subtests.length === 0) {
-      stat.all++;
-    } else {
-      for (const sub of test.subtests) {
-        stat.all++;
-        if (sub.status === 'PASS') {
-          stat.pass++;
-        }
-      }
-    }
-  }
-
-  return `
-    <!DOCTYPE html>
-    <html lang="en">
-    <meta charset="utf-8">
-    <title>BiDi-CDP Mapper</title>
-    <style>
-      body { font-family: Roboto, serif; font-size: 13px; color: #202124; }
-      .path { font-family: Menlo, Consolas, Monaco, Liberation Mono, Lucida Console, monospace; line-height: 180%; padding: 5px 18px; margin: 0; }
-      .top { box-shadow: 0 1px 4px rgba(0, 0, 0, 0.15), 0 1px 6px rgba(0, 0, 0, 0.2); border-radius: 8px; margin: auto; padding: 60px; max-width: 1200px; }
-      .test-card { padding-left: 20px; max-width: 1200px; }
-      .divider { margin-left: 20px; height: 1px; background: #a0a0a0; }
-      .non-collapsible-item { padding-left: 27px; }
-      .stat { float: right }
-      .pass { background: #D5F2D7; }
-      .part { background: #F2EDD5; }
-      .fail { background: #F2D7D5; }
-    </style>
-    <div class="top">
-      <h1>WPT test results</h1>
-      <h2>
-        ${stat.pass} / ${stat.all}
-      </h2>
-      <div>
-        ${report.results.map(generateTestReport).join('')}
-      </div>
-    </div>`;
-}
-
-function generateTestReport(test) {
-  const stat = {all: 0, pass: 0};
-  if (test.status !== 'OK' && test.subtests.length === 0) {
-    stat.all++;
-  } else {
-    for (const subtest of test.subtests) {
-      stat.all++;
-      if (subtest.status === 'PASS') {
-        stat.pass++;
-      }
-    }
-  }
-
-  return `
-    <div class="divider"></div>
-    <div class="test-card">
-      <details>
-        <summary class="path ${
-          stat.all === stat.pass ? 'pass' : stat.pass === 0 ? 'fail' : 'part'
-        }">
-          ${escapeHtml(test.test)}
-          <span class="stat" ><b>${stat.pass}/${stat.all}</b></span>
-        </summary>
-        ${test.subtests.map(generateSubtestReport).join('')}
-      </details>
-    </div>`;
-}
-
-function generateSubtestReport(subtest) {
-  return `
-      <div class="divider"></div>
-      <div class="test-card">
-        <p class="non-collapsible-item path ${
-          subtest.status === 'PASS' ? 'pass' : 'fail'
-        }">
-          ${escapeHtml(subtest.name)}
-          <span class="stat"><b>${escapeHtml(subtest.status)}</b></span>
-        </p>
-      </div>
-`;
-}
-
-const result = generateHtml(readReport(getReportPath()));
+const rawReport = readReport(getReportPath());
```

Comment:
I'd line both of these variables

Reply:
done

Reply:
although I left this variables for readability, I guess it's readable enough )